pkgs/misc/cups/libppd.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, cups
+, libcupsfilters
+, ghostscript
+, pkg-config
+, poppler_utils
+, mupdf # mutool
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libppd";
+  version = "2.0b2";
+
+  src = fetchFromGitHub {
+    owner = "OpenPrinting";
+    repo = "libppd";
+    rev = "2c650af8c8826a68c0c0293ab5511701eaac019a";
+    hash = "sha256-4KRJfzK/xeWlXmhUOlvGHsMmKqF6R4e5BiIA+h2UTII=";
+  };
+
+  configureFlags = [
+    "--disable-acroread"
+    "--localstatedir=/var"
+    "--sysconfdir=/etc"
+  ];
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [
+    cups
+    ghostscript
+    poppler_utils
+    mupdf.bin
```

Comment:
You are sure that the bin output is not executed and should go to nativeBuildInputs?

Reply:
I'm reasonably confident these belong in `buildInputs`: 

- The relevant code in `libppd`/`libcupsfilters` was adapted from `cups-filters` (`pkgs/misc/cups/filters.nix`) which also has mupdf etc. in `buildInputs`
- As far as I can tell `libppd`/`libcupsfilters` only use these tools at runtime

Apart from that, I could probably change `mupdf.bin` back to `mupdf` - apparently `bin` is the default output (?) for `mupdf`